### DISCUSSION TITLE
How to make convenient_test fit for dart sdk 3.2.0

### DISCUSSION THREAD

**Discussion by lehmannk** [fzyzcjy/flutter_convenient_test#354] [Q&A]
My project upgraded to dart sdk 3.2.0 and now all of my nice convenient_tests got incompatible. I cannot even run the tests because of compile errors. So I guess an upgrade for sdk 3.2.0 is required for latest apps to be testable. Has someone maybe even started with this? Of not, what is the best approach to start this? Which package to start with, which tools to use. I am still pretty new to flutter/dart and have never done this before.

**Comment by fzyzcjy** [comment]
Firstly, I strongly recommend not to use dart 3.2 now, since it is often the case that the first few patch versions of flutter is not super stable to be used in production environments. In other words, things like flutter x.y.0 / x.y.1 / ... may be not suitable for production, while x.y.3 / x.y.4 / ... are more suitable.

Secondly, I am planning to migrate to make it fit 3.2, no worries! (Ping me if x.y.3 is released and I still do not migrate after a few days)

**Comment by lehmannk** [comment]
Perfect. Thank you :-)

**Reply by fzyzcjy** [reply]
You are welcome!

**Comment by fzyzcjy** [comment]
Related: https://github.com/fzyzcjy/flutter_convenient_test/pull/355
